src/app/dashboard/profile/page.tsx

```diff
@@ -0,0 +1,104 @@
+"use client"
+
+import React, { useState } from 'react';
+
+import "/Users/aibra11/Desktop/CRM-Web-Application/src/app/Profile.css"
```

Comment:
This is a local import and will not work on remote repositories. When working with a team this will not work